kuma/api/v1/views.py

```diff
@@ -300,3 +301,22 @@ def bc_signal(request):
         serializer.save()
         return Response(serializer.validated_data, status=status.HTTP_201_CREATED)
     return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+@never_cache
+@require_GET
+def get_user(request, username):
+    """
+    Returns a JSON response with a small subset of public information if a
+    user with the given username exists, otherwise returns a status code of
+    404. The case of the username is not important, since the collation of
+    the username column of the user table in MySQL is case-insensitive.
+    """
+    fields = ('username', 'title', 'fullname', 'timezone', 'locale')
```

Comment:
Why these? Why not `organization` and `location` as well?

Reply:
No particular reason other than I chose a subset that I thought would be essential. I'm happy to add `organization` and `location`  and others as well.